src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Public/NLHandler.cs

```diff
@@ -68,10 +68,10 @@ public class NL2FxParameters
         public Engine Engine { get; set; }
 
         /// <summary>
-        /// Set the locale that expect the output expression to be.
+        /// Set the locale name that expect the output expression to be.
         /// For example: "vi-VN" or "fr-FR".
         /// </summary>
-        public CultureInfo ExpressionLocale { get; set; }

```

Comment:
>CultureInfo [](http://example.com/codeflow?start=15&length=11)

Why is CultureInfo bad, but Engine on line 68 is ok?

Reply:
We never serialize the Nl2FxParmaters. This object is transferred to Nl2FxPayload so CultureInfo is transferred as it is. Nl2FxPayload's CultureInfo is serialized in the end. This is just the DTO. There's no need to change this. To answer your question, we never serialize Engine

Reply:
ok so i only need to change in internal repo. I thought i need to change in both here and internal.

Reply:
no you don't need this pr at all. also in general we should mostly avoid serializing built in types other than mostly collections, and primitive types or the ones which has explicit support for serialization. please add a comprehensive set of tests in Nl library testing the serialization of Nl2FxPayload to prevent this